src/test/java/com/wikia/webdriver/testcases/mercurytests/SEOTests.java

```diff
@@ -112,7 +112,7 @@ public void MercurySEOTest_001_MetaTags_CanonicalLink() {
     /*String lastDesc = seo.getDescription();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)